submissions/bettyBada/html-forms/product.html

```diff
@@ -34,9 +34,11 @@ <h1 id="logo" class="highlight">Asterion Gadgets</h1>
             <h2 class="prod-name">BLUE-WHITE COLLAR TEE SHIRT</h2>
             <p class="brand">Brand: Polka-dot</p>
             <span class="quantity">Quantity:</span>
-            <input type="text" class="box">
-            <p class="price">Price: $150</p>
-            <input class="done" type="submit" value="BUY NOW">
+            <input type="number" class="box" value="1" min="1" id="quantity" onchange="total_cost()">
```

Comment:
Are you sure this input field listens for an `onchange` event? I only know `select` tags to listen for this event

Reply:
Yes, it does. An input field can listen for an onchange event.